pkg/apis/api-rules/violation_exceptions.list

```diff
@@ -1 +1,3 @@
 API rule violation: names_match,github.com/argoproj/argo-rollouts/pkg/apis/rollouts/v1alpha1,RolloutStatus,HPAReplicas
+API rule violation: names_match,github.com/argoproj/argo-rollouts/pkg/apis/rollouts/v1alpha1,ScopeDetail,EndTime
+API rule violation: names_match,github.com/argoproj/argo-rollouts/pkg/apis/rollouts/v1alpha1,ScopeDetail,StartTime
```

Comment:
Ensure that StartTime and EndTime match the json struct tags in order to remove these violations